frontends/bmd/src/__snapshots__/app_contest_candidate_no_party.test.tsx.snap

```diff
@@ -479,7 +479,7 @@ exports[`Single Seat Contest 1`] = `
     class="c1"
   >
     <main
-      class="sc-bkkfTU enzCGe"
+      class="sc-cxpRKc iyrZoO"
```

Comment:
This simple rename seems to have caused a lot of snapshot updates, as documented in issue https://github.com/votingworks/vxsuite/issues/1617